x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_events_query.ts

```diff
@@ -208,7 +208,7 @@ export const buildEventsSearchQuery = ({
   return searchQuery;
 };
 
-export const buildEqlSearchRequest = ({
+export const buildEqlSearchRequest = async ({
```

Comment:
When `getQueryFilter` is changed back to sync, this won't need to be async either